bedrock/settings/__init__.py

```diff
@@ -95,22 +95,32 @@
 
 site_mode = config("SITE_MODE", default=MOZORG_SITE_MODE)
 
-if site_mode == POCKET_SITE_MODE:
+IS_POCKET_MODE = site_mode == POCKET_SITE_MODE
+IS_MOZORG_MODE = not IS_POCKET_MODE
+
+if IS_POCKET_MODE:
     ROOT_URLCONF = "bedrock.urls.pocket_mode"
 
-    # DROP the redirects app and middleware, because it's contains Mozorg-specific rules that
-    # clash with some Pocket URL paths (eg /jobs/)
+    # DROP the redirects app and middleware, because its contains Mozorg-specific
+    # rules that clash with some Pocket URL paths (eg /jobs/)
     INSTALLED_APPS.pop(INSTALLED_APPS.index("bedrock.redirects"))
     MIDDLEWARE.pop(MIDDLEWARE.index("bedrock.redirects.middleware.RedirectsMiddleware"))
 
     # TODO: define in Pocket-appropriate versions of
     # DEV_LANGUAGES, PROD_LANGUAGES, CANONICAL_LOCALES,
-    # FLUENT_* overrides for Pocket L10N, etc
+    FLUENT_DEFAULT_FILES = [
+        "brands",
+        "nav",
+        "footer",
+    ]
+
+    # Swap the default FLUENT_LOCAL_PATH for a Pocket-specific one
+    FLUENT_PATHS.pop(FLUENT_PATHS.index(FLUENT_LOCAL_PATH))
+    FLUENT_LOCAL_PATH = ROOT_PATH / "l10n-pocket"
+    FLUENT_PATHS.insert(0, FLUENT_LOCAL_PATH)
```

Comment:
I'd still rather us just fully redefine `FLUENT_PATHS` here since the whole thing will indeed be different once we get the l10n-team's repo setup and syncing.

Reply:
yeah - fixing